hadoop-ozone/interface-storage/src/main/java/org/apache/hadoop/ozone/om/OMMetadataManager.java

```diff
@@ -260,13 +261,14 @@ List<OmVolumeArgs> listVolumes(String userName, String prefix,
    * Returns the names of up to {@code count} open keys whose age is
    * greater than or equal to {@code expireThreshold}.
    *
-   * @param count The maximum number of open keys to return.
+   * @param limit The maximum number of open keys to return.
    * @param expireThreshold The threshold of open key expire age.
-   * @return a list of {@link String} representing names of open expired keys.
+   * @param bucketLayout The type of open keys to get (e.g. DEFAULT or FSO).
+   * @return a {@link List} of {@link OpenKeyBucket}, the expired open keys.
```

Comment:
```suggestion
   * @return a {@link List} of {@link OpenKeyBucket}: List of open keys per bucket.
```

Reply:
Feels redundant to me, and it breaks line line limit.